### PR TITLE
Space Travelers' Hub final pull request

### PR DESCRIPTION
**Pull request by SebastianSolisFenger**
## Changes in this PR

- [x] - Add functions to fetch data from API and update Redux store
- [x] - Implement fetching data from API only on the first render of the application
- [x] - Add actions, action creators, and reducers for fetching, getting rockets and missions, booking rockets, and joining missions, actions, and reducers for canceling rockets and leaving missions
- [x] - Create the Rockets, Missions, and Profile pages based on the provided design
- [x] - Write tests

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,23 @@
+import { Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+// import './index.css';
+// import Missions from './components/Missions';
+// import AllRockets from './components/AllRockets';
```
Comment (prolajumokeoni) [inline src/App.js]: - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  return (
+    <div className="App">
+      <Navbar />
+      <Routes>
+        {routes.map((route) => (
+          <Route path={route.path} element={route.element} key={route.id} />
+        ))}
+      </Routes>
+      {/* <AllRockets />
+      <Missions /> */}
```
Comment (prolajumokeoni) [inline src/App.js]: - [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Team,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @Team,

You've done well addressing concerns from the previous review 👏🏼.
Tasks for this project are now successfully implemented 💯.

* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* App deployed and available online ✔️ 
* In the project:
   * Good usage of the react-router-dom package ✔️ 
   * Clean codebase ✔️ 
   * Redux store and redux logic properly implemented ✔️  
   * All expected features of the MVP functioning as expected ✔️ 

All the best with the next project.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**